implementations/rust/ockam/ockam_command/src/terminal/mod.rs

```diff
@@ -151,13 +152,13 @@ impl<W: TerminalWriter> Terminal<W> {
     fn should_disable_color(no_color: bool) -> bool {
         // If global argument `--no-color` is passed or the `NO_COLOR` env var is set, colors
         // will be stripped out from output messages. Otherwise, let the terminal decide.
-        no_color || std::env::var("NO_COLOR").is_ok()
+        no_color || get_env("NO_COLOR", false).unwrap_or_else(|e| false)
```

Comment:
I think I would just error out in that case. If a variable is defined but cannot be parsed when starting the command line I think it is better to inform the user right away.